src/render/hud.rs

```diff
@@ -521,16 +521,13 @@ impl Screen for Hud {
 
 impl Hud {
     pub fn icon_scale(renderer: Arc<Renderer>) -> f64 {
-        Hud::icon_scale_by_height(renderer.screen_data.read().safe_height)
+        let screen = renderer.screen_data.read();
+        Hud::icon_scale_by_height(screen.safe_width, screen.safe_height)
     }
 
-    pub fn icon_scale_by_height(height: u32) -> f64 {
-        let icon_scale = if height > 500 {
-            height as f64 / 36.50
-        } else {
-            height as f64 / 26.50 /*27.5*/
-        };
-        icon_scale / 9.0
+    pub fn icon_scale_by_height(width: u32, height: u32) -> f64 {
```

Comment:
this is now scaled by width and height, the function name should be changed accordingly to something like `icon_scale_by_dims` or smth like that